### DISCUSSION TITLE
essay/2024/11/08/fame

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#244] [Comments]
# essay/2024/11/08/fame

얼굴이 알려지면 얼마나 생활이 불편할까? 유명해지지 않도록 항상 조용히 살아야지.

https://jeho.page/essay/2024/11/08/fame.html

**Comment by junah201** [comment]
밋업 신청했는데 당첨되서 꼭 반갑게 인사할 수 있으면 좋겠네요 ㅎ

**Reply by BenjaminKim** [reply]
@junah201 당첨 꼭 되셨으면 좋겠습니다! 😁

**Comment by synuns** [comment]
솔로프리너 취소표를 기다립니다!!

**Reply by synuns** [reply]
재호님! 어제 솔로프리너 강연 정말 잘들었습니다 ㅎㅎ
인사는 따로 못드려서 너무 아쉬웠지만, 새로운 시작의 원동력이 될 것 같습니다!
감사드립니다 ㅎㅎ

**Reply by BenjaminKim** [reply]
그러셨군요. 저도 못 뵈서 아쉽습니다.
새로운 시작 응원하겠습니다. 💪

**Comment by yoonjong-park** [comment]
인사드리겠습니다. 마곡에서 뵈여요ㅎㅎ

**Reply by yoonjong-park** [reply]
어제 솔로프리너 강연 보고 왔습니다.
저도 이제 수익 250 안팎정도 매달 나옵니다.
나중에 회사월급 넘어가서 어른이 되면 메일 보낼게요ㅎㅎ
감사합니다.

**Reply by BenjaminKim** [reply]
@yoonjong-park 250 매달 나오면 이미 성공하신 것 아닌가요? (엄청 대단한 겁니다)
정말 축하드립니다!
